### DISCUSSION TITLE
New music thread

### DISCUSSION THREAD

**Discussion by mattl** [foocorp/hacienda#88] [New Music]
It's been a while, but I'd love to hear some new music from anyone releasing stuff under a Creative Commons Attribution or Attribution-ShareAlike license. If you have anything to suggest, please leave it as a comment with a link to buy/stream/download.

**Comment by Calinou** [comment]
[Marc A. Pullen](https://marcapullen.bandcamp.com/) would be a good addition (all albums are CC BY-SA).

[Cloudkicker](https://cloudkicker.bandcamp.com/music) has all their albums uploaded before 2020 under CC BY, but albums uploaded after that are CC BY-NC-SA.
